src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/DefaultRazorSemanticTokensInfoService.cs

```diff
@@ -8,16 +8,19 @@
 using Microsoft.AspNetCore.Razor.LanguageServer.Common;
 using Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models;
 using Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Services;
+using Microsoft.CodeAnalysis.Razor;
```

Comment:
This shouldn't cause additional DLLs to load in non-LSP powered cases right?

Reply:
If  the  Microsoft.CodeAnalysis.Razor  assembly isn't already loaded in nonLSP scenarios then yes it will. Sooo it  depends on if we're  already loading it which I thinkkkkk we are

Reply:
Merged this in, will insert and see if it results in additional DLLs loaded. May have to revert.